src/StructuredDiscussions/SDTopic.php

```diff
@@ -112,6 +112,15 @@ public function getModerationState(): string {
 		return $this->getRootRevision()['moderateState'] ?? AbstractRevision::MODERATED_NONE;
 	}
 
+	/**
+	 * Returns the topic's current lock state.
+	 *
+	 * @return string
+	 */
+	public function getLockStatus(): string {
```

Comment:
This should be `public function getLockStatus(): bool` to support booleans.

Reply:
Done